src/frontend/Komponenter/Behandling/SettPåVent/LokalkontorOppgavevalg.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { FC, useState } from 'react';
+import { CheckboxGroup, Checkbox, BodyShort } from '@navikt/ds-react';
+import styled from 'styled-components';
+import { dagensDatoFormatert, formaterIsoDato } from '../../../App/utils/formatter';
+
+const FlexBox = styled.div`
+    display: flex;
+    align-items: center;
+    gap: 0.5rem;
+`;
+
+export type Oppgavestatus = {
+    vurderHenvendelseOppgave: VurderHenvendelseOppgavetype;
+    datoOpprettet: string;
+};
+
+type Props = {
+    aktuelleOppgaver: VurderHenvendelseOppgavetype[];
+    oppgavestatus: Oppgavestatus[];
+    settOppgaverMotLokalkontor: (oppgaverMotLokalkontor: VurderHenvendelseOppgavetype[]) => void;
```

Comment:
Dette blir litt mer presist da vi også kan ta inn prevstate i en callback 🤓 
```suggestion
    settOppgaverMotLokalkontor: Dispatch<SetStateAction<VurderHenvendelseOppgavetype[]>>
```